includes/class-wc-siftscience-admin.php

```diff
@@ -256,86 +253,69 @@ public function add_settings_page( $pages ) {
+				$this->get_element( 'number', WC_SiftScience_Options::THRESHOLD_BAD,
+					'Bad Score Threshold', 'Scores above this value are considered bad and shown in red', array( 'default' => 60 ) ),
 
-				$this->get_text_input( WC_SiftScience_Options::NAME_PREFIX,
+				$this->get_element( 'text', WC_SiftScience_Options::NAME_PREFIX,
 					'User & Order Name Prefix',
 					'Prefix to give order and user names. '
 					. 'Useful when you have have multiple stores and one Sift account.' ),
 
-				$this->get_check_box( WC_SiftScience_Options::AUTO_SEND_ENABLED,
+				$this->get_element( 'checkbox', WC_SiftScience_Options::AUTO_SEND_ENABLED,
 					'Automatically Send Data',
 					'Automatically send data to Sift when an order is created'
 				),
 
-				$this->get_number_input( WC_SiftScience_Options::MIN_ORDER_VALUE,
+				$this->get_element( 'number', WC_SiftScience_Options::MIN_ORDER_VALUE,
 					'Minimum Order Value for Auto Send',
-					'Orders less than this value will not be automatically sent to sift. Set to zero to send all orders.', 0 ),
+					'Orders less than this value will not be automatically sent to sift. Set to zero to send all orders.', array( 'default' => 0 ) ),
 
-				$this->get_section_end( 'sifsci_section_main' ),
+				$this->get_element( 'sectionend','sifsci_section_main' ),
 			);
 		}
 
-		private function get_title( $id, $title ) {
-			return array( 'title' => $title, 'type' => 'title', 'desc' => '', 'id' => $id );
-		}
-
-		private function get_text_input( $id, $title, $desc ) {
-			return array(
-				'title' => $title,
-				'desc' => $desc,
-				'desc_tip' => true,
-				'type' => 'text',
-				'id' => $id,
-			);
-		}
+		private function get_element( $type, $id, $title = '', $desc = '', $special_options = array() ) {
+			
+			$element = array( 'type' => $type, 'id' => $id );
+
+			switch ( $type ) {
+				case 'sectionend':
+					return $element;
+				case 'title':
+					return array_merge( $element, array( 'title' => $title, 'desc' => $desc ) );
+				case 'number':
+				case 'select':
+					if ( ! empty( $special_options ) ) {
+
+						$element = array_merge( $element, $special_options );
+					}
+				case 'checkbox':
+				case 'text':	
+					if ( ! empty( $desc ) ) {
 
-		private function get_number_input( $id, $title, $desc, $default ) {
-			return array(
-				'id' 		=> $id,
-				'title' 	=> $title,
-				'desc' 		=> $desc,
-				'default' 	=> $default,
-				'type' 		=> 'number',
-				'desc_tip' 	=> true
-			);
-		}
+						$element = array_merge( $element, array( 'desc' => $desc, 'desc_tip' => true ) );
 
-		private function get_section_end( $id ) {
-			return array( 'type' => 'sectionend', 'id' => $id );
-		}
+					}
 
-		private function get_check_box( $id, $title, $desc ) {
-			return array(
-				'title' => $title,
-				'desc' => $desc,
-				'desc_tip' => true,
-				'type' => 'checkbox',
-				'id' => $id,
-			);
-		}
+					$element['title'] = $title;
+					break;
+				default:
+					error_log( $type . ' is nut supported yet!' );
```

Comment:
What kind of nut? Almond or pistachio? 
:-D

Also: "yet" is probably wrong wording. If it's a typo, the unknown type will never be supported.

Reply:
No clue what err msg

Reply:
Write what you prefer